### ISSUE TITLE
After File edit old commit is loaded

### ISSUE THREAD

**Issue by JuKu** [k0shk0sh/FastHub#1809]
**FastHub Version: 4.6.7**  
**Android Version: 7.0 (SDK: 24)**  
**Device Information:**  
- **Manufacturer:** motorola  
- **Brand:** motorola  
- **Model:** Moto G (5) Plus  
---

 If i change a file, save it and open it again, Changes has no affect (old file is loaded).

**Comment by k0shk0sh** [comment]
The file is cached from GitHub it takes some time for it to load

**Comment by JuKu** [comment]
Thanks for your answer!
Maybe a warning "you see old file from cache, loading..." will be a good thing?

But if i edit again, also old commit is loaded. So i dont edit the current commit, i edit an older one.

**Comment by k0shk0sh** [comment]
yes exactly and there is no way for me to see know that this is old or new, i display what GitHub returns. 🤷‍♂️

**Comment by JuKu** [comment]
Do you use the github api or a library?

**Comment by yakov116** [comment]
@JuKu  api

**Comment by JuKu** [comment]
And the api caches this output?
Is there a opportunity to avoid caching? Maybe a parameter on url?

**Comment by yakov116** [comment]
@JuKu yup, we can't fix GitHub's problems
